pkg/generate/app/cmd/newapp.go

```diff
@@ -862,7 +876,7 @@ func (c *AppConfig) Run() (*AppResult, error) {
 // RunQuery executes the provided config and returns the result of the resolution.
 func (c *AppConfig) RunQuery() (*QueryResult, error) {
 	c.ensureDockerSearcher()
-	repositories, err := c.individualSourceRepositories()
+	_, err := c.individualSourceRepositories()
```

Comment:
?

"repositories" wasn't being referenced anywhere, that's why i dropped it.  (it was just getting overwritten on line 880).